recipes/reaktoro/bld.bat

```diff
@@ -0,0 +1,23 @@
+mkdir build
+cd build
+
+REM This is a fix for a CMake bug where it crashes because of the "/GL" flag
+REM See: https://gitlab.kitware.com/cmake/cmake/issues/16282
+@echo CXXFLAGS=%CXXFLAGS%
+@echo CFLAGS=%CFLAGS%
+set CXXFLAGS=%CXXFLAGS:-GL=%
+set CFLAGS=%CFLAGS:-GL=%
+@echo After removing -GL flag because of CMake bug:
+@echo CXXFLAGS=%CXXFLAGS%
+@echo CFLAGS=%CFLAGS%
+
+cmake -G Ninja ^
+      -DBUILD_ALL=ON ^
+      -DCMAKE_BUILD_TYPE=Release ^
+      -DCMAKE_INCLUDE_PATH="%LIBRARY_INC%" ^
+      -DBOOST_INCLUDE_DIR="%LIBRARY_INC%" ^
+      -DCMAKE_INSTALL_PREFIX="%LIBRARY_PREFIX%" ^
+      -DCMAKE_VERBOSE_MAKEFILE:BOOL=ON ^
+      "-DTHIRDPARTY_COMMON_ARGS=-DCMAKE_VERBOSE_MAKEFILE:BOOL=ON" ^
+      ..
+cmake --build . --config Release --target install
```

Comment:
Please use ninja to build

Reply:
Why? It is using Ninja already under the hood ;)

Reply:
I know. The problem is I'm trying to reduce the use of cmake in recipes because I just saw it do not work well with sysroot, and sometimes cmake does not provide to you the whole control of the building process.

But the thing in this case is, you are calling cmake and cmake is calling ninja and ninja will start the building process. Another thing which is funny is that you can specify the release in the cmake configuration and also specified in the build.
Just calling ninja it is easy to understand as well.

``cmake --build . --config Release --target install`` vs ``ninja install``

Reply:
It seems to be a matter of taste in this case, I prefer `cmake --build` because it is standardized by CMake itself, no matter what generator/build system is chosen, the command is always the same.

It is also recommended as a good practice when using CMake, see this slide: https://www.slideshare.net/DanielPfeifer1/cmake-48475415/17

But no problem, I can change that